src/main/java/org/ecommerce/Main.java

```diff
@@ -102,6 +106,5 @@ public static void main (String[] args) {
         System.out.println(orderController.create(request));
```

Comment:
Please use the `Logger` class and remove all `System.out.println` calls

Reply:
Here is the commit https://github.com/algutierrez-griddynamics/e-commerce/commit/0230163bcca63bf5981857e8427e4f72e73b2337